gui/widgets/bandview.cpp

```diff
@@ -330,27 +386,30 @@ void BandView::cursorAction(QGraphicsSceneMouseEvent *ev, bool click)
 		emit pixelOverlay(cursor.y(), cursor.x());
 
 	/// single label case
-	if (singleLabel && showLabels) {
-		short cursorLabel = labels(cursor.y(), cursor.x());
-		if (ev->buttons() & Qt::LeftButton) {
-			holdLabel = !holdLabel;
-		}
-		if (!holdLabel && (cursorLabel != curLabel)) {
-			curLabel = cursorLabel;
-			curMask = cv::Mat1b(labels.rows, labels.cols, (uchar)0);
-			curMask.setTo(1, (labels == curLabel));
-			drawOverlay(curMask);
-			emit singleLabelSelected(curLabel); // dist view redraw
-		} else {
-			if (overlay != &curMask)
-				drawOverlay(curMask);
-		}
+    if (sm == Pick && showLabels) {
+
+
```

Comment:
I think there is a misconception here. Like in the LabelDock, here we should also be able to select multiple labels. Also the holdLabel doesn't make sense anymore. Just select and deselect (additional) labels with left click.